### ISSUE TITLE
Runtime: The cigarette attempted to add a reagent called 'menthol' which doesn't exist.

### ISSUE THREAD

**Issue by Neerti** [PolarisSS13/Polaris#6806]
#### Brief description of the issue
A runtime error occurs on server initialization, quite frequently due to the large number of cigs mapped in.

#### What you expected to happen
No runtimes to happen before the round even starts.

#### What actually happened
```
[2020-03-09T09:15:03] Runtime in unsorted.dm,1315: The cigarette attempted to add a reagent called 'menthol' which doesn't exist. ()
  proc name: crash with (/proc/crash_with)
  src: null
  call stack:
  crash with("The cigarette attempted to add...")
  /datum/reagents (/datum/reagents): add reagent("menthol", 2, null, 0)
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol), /list (/list))
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): New(1)
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): New(the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols))
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): New(the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols))
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): New(the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols), null)
  the cigarette (/obj/item/clothing/mask/smokable/cigarette/menthol): New(the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols))
  the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols): Initialize(1)
  the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols), /list (/list))
  the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols): New(1)
  the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols): New(the plating (148,150,1) (/turf/simulated/floor/plating))
  the pack of Temperamento Menth... (/obj/item/weapon/storage/fancy/cigarettes/menthols): New(the plating (148,150,1) (/turf/simulated/floor/plating))
  the random cigarettes (/obj/random/cigarettes): spawn item()
  the random cigarettes (/obj/random/cigarettes): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the random cigarettes (/obj/random/cigarettes), /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms(null)
  Atoms (/datum/controller/subsystem/atoms): Initialize(476706)
  Master (/datum/controller/master): Initialize(10, 0)
```
#### Steps to reproduce
Start up a local instance and watch the runtimes flow in.

#### Additional info:
 - **Server Revision**: Found using the "Show Server Revision" verb under the OOC tab.
Server revision: master - 2020-03-09
a4100b5b74b9f850f3e5391bb08582e05128bce8 
 - **Anything else you may wish to add** (Location if it's a mapping issue, etc)
This is almost certainly from #6753 
This appears to be caused by the referenced reagent lacking the `id` var being set, if so it should be an easy fix.

**Comment by Cerebulon** [comment]
Got a fix up for this, #6797 , the PR forgot to define an identifier for menthol when they added it

**Comment by Atermonera** [comment]
#6797 cerebul you're supposed to put `fixes x` in the pr so I don't have to go through these manually and see if they've been fixed by PRs I remember seeing in the recent past
